test/e2e/framework.go

```diff
@@ -82,7 +82,7 @@ const (
 	antreaDeployment           string = "antrea-controller"
 	flowAggregatorDeployment   string = "flow-aggregator"
 	antreaDefaultGW            string = "antrea-gw0"
-	testNamespace              string = "antrea-test"
+	//testNamespace              string = "antrea-test"
```

Comment:
Remove a line when it's no longer needed

Reply:
sorry my bad i missed it.